package.json

```diff
@@ -127,13 +127,13 @@
         "webpack-pwa-manifest": "^4.2.0"
     },
     "dependencies": {
-        "@hcaptcha/react-hcaptcha": "^0.3.6",
         "@dnd-kit/core": "^3.1.1",
         "@dnd-kit/modifiers": "^3.0.0",
         "@dnd-kit/sortable": "^4.0.0",
         "@dnd-kit/utilities": "^2.0.0",
+        "@hcaptcha/react-hcaptcha": "^0.3.6",
         "@mapbox/mapbox-gl-draw": "^1.2.0",
-        "@the-deep/deep-ui": "1.4.2-beta.0",
+        "@the-deep/deep-ui": "^1.4.2",
```

Comment:
What does the '^' mean?

Reply:
~version “Approximately equivalent to version”, will update you to all future patch versions, without incrementing the minor version. ~1.2.3 will use releases from 1.2.3 to <1.3.0.

^version “Compatible with version”, will update you to all future minor/patch versions, without incrementing the major version. ^2.3.4 will use releases from 2.3.4 to <3.0.0.